packages/modeling/src/operations/extrusions/earcut/polygonTrees.js

```diff
@@ -0,0 +1,43 @@
+const geom2 = require('../../../geometries/geom2')
+const OrthoNormalBasis = require('../../../maths/OrthoNormalBasis')
```

Comment:
UG... I'm trying to get rid of OrthoNormalBasis. It's really untested and full of junk.
I know that it's still being used but only by booleans, but not for long.

If useful, can you break out the useful pieces just for earcut? Can something else be used?

Reply:
done. I only needed a couple line from OrthoNormalBasis, and they are pretty simple vector operations. I moved them into polygonTrees.